huxley/www/views.py

```diff
@@ -21,6 +22,7 @@ def index(request):
 
     context = {
         'user_json': json.dumps(user_dict).replace('</', '<\\/'),
+        'conference_session': Conference.objects.filter(session=64)[0].session,
```

Comment:
Seems like an unnecessary fetch, since we're hardcoding the session to `64`. Can we just get the latest conference by session instead? That way it'll work for whatever conference we're at.